src/test/java/org/isf/exam/rest/ExamRowControllerTest.java

```diff
@@ -0,0 +1,246 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.isf.exam.rest;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.log;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+import org.isf.OpenHospitalApiApplication;
+import org.isf.exa.manager.ExamBrowsingManager;
+import org.isf.exa.manager.ExamRowBrowsingManager;
+import org.isf.exa.model.Exam;
+import org.isf.exa.model.ExamRow;
+import org.isf.exam.data.ExamHelper;
+import org.isf.exam.mapper.ExamMapper;
+import org.isf.exam.mapper.ExamRowMapper;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Exam Row Controller Test
+ * @author Silevester D.
+ */
+@SpringBootTest(classes = OpenHospitalApiApplication.class)
+@AutoConfigureMockMvc
+public class ExamRowControllerTest {
+	@Autowired
+	private MockMvc mvc;
+
+	@Autowired
+	private ExamRowMapper mapper;
+
+	@Autowired
+	private ExamMapper examMapper;
+
+	@MockBean
+	private ExamBrowsingManager examManager;
+
+	@Autowired
+	private ObjectMapper objectMapper;
+
+	@MockBean
+	private ExamRowBrowsingManager manager;
+
+	@Test
+	@WithMockUser(username = "admin", authorities = {"examrows.read"})
+	@DisplayName("Should get all exam rows")
+	void testGetAll() throws Exception {
+		var exam = examMapper.map2Model(ExamHelper.generateExam());
```

Comment:
I think this has been pointed about previously.   Even though `var` is valid, the project wants to use strictly typed objects.